_posts/2018/2018-10-14-a-scope-should-return-a-scope.md

```diff
@@ -0,0 +1,55 @@
+---
+title: A scope should return a scope
+description: 'Simple code organisation can sometimes be enough.'
+layout: article
+category: ruby
+image:
+  base: '2018/a-scope-should-return-a-scope'
+  alt: 'Coin-operated (tele)scope'
+  source: 'https://unsplash.com/photos/K40awqK-hTI'
+  credit: 'Laurenz Kleinheider'
+---
+
+The more you can stay on the ‘rails’ when coding Ruby on Rails application the easier you life will be when maintaining the apps you’re building.
+
+A good way of doing this is to try and stick to the patterns set out in the standard Rails APIs.
```

Comment:
Do you want to include anything about scopes in your intro? I think the intros are most effective when you introduce the topic a little and make people wonder: oh yeah, what is the right thing to do in this case? Or when you introduce the specific pain point.

What do you think?